src/pageTransition.js

```diff
@@ -481,8 +488,9 @@ async function initialize() {
                     pageId: eventUpdateMessage.pageId,
                     url: eventUpdateMessage.url,
                     referrer: eventUpdateMessage.referrer,
+                    tabId: sender.tab.id,
                     isHistoryChange: eventUpdateMessage.isHistoryChange,
-                    isOpenedTab: eventUpdateMessage.isOpenedTab,
+                    openerTabId: eventUpdateMessage.openerTabId,
```

Comment:
For clarity, would explicitly handle the case where `openerTabId` isn't present.
```suggestion
                    openerTabId: "openerTabId" in eventUpdateMessage ? eventUpdateMessage.openerTabId : browser.tabs.TAB_ID_NONE,
```